package.json

```diff
@@ -5,28 +5,32 @@
   "description": "simple starter",
   "main": "index.js",
   "scripts": {
-    "start": "npm run concurrent",
-    "concurrent": "concurrently --kill-others \"npm run dev:js\" \"npm run dev:css\" \"npm run dev:server\"",
-    "bundle": "concurrently \"rollup -c\" \"postcss app/index.css -o app/bundle.css\"",
     "dev:js": "rollup -c -w",
     "dev:css": "postcss app/index.css -o app/bundle.css -w",
-    "dev:server": "browser-sync start --server 'app' --files 'app/index.html,app/bundle.css,app/bundle.js' --no-open"
+    "dev:server": "browser-sync start --server 'app' --files 'app/index.html,app/bundle.css,app/bundle.js' --no-open",
+    "build:js": "rollup -c --environment NODE_ENV:production",
+    "build:css": "postcss app/index.css -o dist/bundle.css",
+    "build:html": "cp app/index.html dist/index.html",
+    "build": "run-s build:*",
+    "start": "concurrently --kill-others npm:dev:*"
   },
   "license": "ISC",
-  "dependencies": {},
-  "devDependencies": {
-    "browser-sync": "^2.26.4",
-    "concurrently": "^4.1.0",
+  "dependencies": {
```

Comment:
looks like devDeps and deps got mixed up. can you drop all these back into dev deps please?

Reply:
Sure… I split them like this on the basis of “what would a build require if you set `NODE_ENV=production`“ (Ie. `devDependencies` aren’t installed).
Don’t mind reverting either way.